services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/service/NodeService.java

```diff
@@ -0,0 +1,22 @@
+package com.futurewei.alcor.nodemanager.service;
+
+import com.futurewei.alcor.nodemanager.entity.NodeInfo;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+
+public interface NodeService {
+    int getNodeInfoFromFile(String strPath) throws Exception;
+
+    int getNodeInfoFromUpload(MultipartFile file) throws Exception;
```

Comment:
Remove getNodeInfoFromUpload as not in use.

Reply:
It is used. This is the function which upload machine.json and add bulk node meta data.